pkg/controller/subnet.go

```diff
@@ -289,7 +289,7 @@ func formatSubnet(subnet *kubeovnv1.Subnet, c *Controller) error {
 		subnet.Spec.Provider = util.OvnProvider
 		changed = true
 	}
-	if subnet.Spec.Protocol == "" || subnet.Spec.Protocol != util.CheckProtocol(subnet.Spec.CIDRBlock) {
```

Comment:
The protocol field can be filled automatically if it is not set. The subnet crd should be created as easy as posible.

Reply:
if  i  create or update a error  ip  protocol(like  aaa )， subnet  crd help me changeing it to correct，i think it's not good idea. 
  If ip protocol filed is only be to demonstration  when someone  passes the  command  "kubectl get subnet "?  
I think it should be in the status  zone.

Reply:
Are there some situations in your environment to set the protocol field(correct or wrong)，and is used specially ?

Reply:
yes,  We support creating subnets using YAML. ApiServer  need to check whether the protocol fields are correct.